examples/bert/src/bert_layers.py

```diff
@@ -545,6 +546,70 @@ def forward(
             all_encoder_layers.append(hidden_states)
         return all_encoder_layers
 
+    # Param Initialization, needed for device='meta' fast initialization
+    def param_init_fn(self, module):
```

Comment:
@jacobfulano a few comments:
* `param_init_fn` is not needed unless you plan to make your models `meta` initializable. For a first draft, I would remove entirely
* `fsdp_wrap_fn` and `activation_checkpointing_fn` need to be defined on a root module underneath the `ComposerModel`. So they have to be defined for say, `BertForMaskedLM`. I would recommend moving the definition there, and I think it will work!